test/unit/helpers/rbuffer.h

```diff
@@ -0,0 +1,9 @@
+#include "nvim/rbuffer.h"
```

Comment:
I think the more descriptive name in testing parlance is "fixtures". In the functional tests we have `test/functional/fixtures/`, so this new path might as well be renamed to `test/unit/fixtures/`.